app/femr/ui/models/triage/IndexViewModelPost.java

```diff
@@ -55,6 +58,13 @@
     //indicates if the "yes" button was clicked for the diabetes screening prompt
     private String isDiabetesScreenPerformed;
 
+    public List<ValidationError> validate(){
```

Comment:
This causes a global error instead of handling it gracefully, which is better than nothing because it doesn't end up in the database, I guess, but since it's triggerable without much difficulty then I think it's an issue.

It also only checks against letters, whereas the regex client-side removes the first non-digit. Meaning you can send any symbol and it'll gladly be put into the database.